SPEC/STATS.md

```diff
@@ -25,7 +25,9 @@ Where:
   - `poll` is used to print bandwidth at an interval.
   - `interval` is the time interval to wait between updating output, if `poll` is true.
 
-`callback` must follow `function (err, stats) {}` signature, where `err` is an error if the operation was not successful. `stats` is an Object containing the following keys:
+If `poll` is `true`, then `callback` must follow `function (err, stream) {}` signature, where `err` is an error if the operation was not successful and `stream` is a Readable Stream where you can listen to the event `data` with a listener that must follow `function (stat) {}` signature.
+
+Otherwise, `callback` must follow `function (err, stat) {}` signature, where `err` is an error and `stat` is an Object containing the following keys:
```

Comment:
We should define what the values are as well, I *think* the values are defined in bytes, but unsure.

Reply:
? The `stat` thing on the stream?

Reply:
If so, it is an object like the other one. That's why I left them with the same name and explained in the end.

Reply:
Yeah, that `stat` object. We say which keys are on the object, but not what the values are.

Reply:
Problem is not understanding that `stat` is used in both cases, but what the values are. Are they bytes? Are they kilobytes?

Reply:
I'll add that!